packages/cli/src/commands/organization-services/login.ts

```diff
@@ -0,0 +1,44 @@
+import { OrganizationServicesService } from '@meeco/sdk';
+import { AuthConfig } from '../../configs/auth-config';
+import { authFlags } from '../../flags/auth-flags';
+import MeecoCommand from '../../util/meeco-command';
+
+export default class OrganizationServicesLogin extends MeecoCommand {
+  static description =
+    'Login as a service agent. An service agent is a non-human Vault user account acting on behalf of the service. An organization owner or admin can use this command to obtain a session token for the service agent.';
```

Comment:
Wrap in backticks so this can line wrap (line is a bit too long)

Reply:
done